hardware/adreno/Android.mk

```diff
@@ -28,4 +28,9 @@ SONY_SYMLINKS := $(foreach lib_dir,lib lib64, \
     ) \
 )
 
+# Special exception for vulkan.qcom.so that is also linked as vulkan.$(TARGET_BOARD_PLATFORM).so
+SONY_SYMLINKS := $(foreach lib_dir,lib lib64, \
```

Comment:
`:=` overwrites the previous value of `SONY_SYMLINKS`, meaning that all other links defined above are lost. Please change it to `+=`.

Reply:
Also, thanks @erikcas for noticing the missing backslash on the next line.